pkg/slo-controller/noderesource/plugins/midresource/plugin.go

```diff
@@ -161,6 +200,17 @@ func (p *Plugin) calculate(strategy *configuration.ColocationStrategy, node *cor
 	}
 	memory := resource.NewQuantity(allocatableMemory, resource.BinarySI)
 
+	// add unallocated
+	unallocated := p.getUnallocated(node, podList)
+	// CPU need turn into milli value
+	unallocatedCPU, unallocatedMemory := resource.NewQuantity(unallocated.Cpu().MilliValue(), resource.DecimalSI), unallocated.Memory()
```

Comment:
how about `unallocatedMilliCPU` or `unallocatedMilliCores`, let's make the naming concise

Reply:
pretty good, fixed.